src/Utilities/Requires.hpp

```diff
@@ -11,7 +11,7 @@
 namespace Requires_detail {
 template <bool B>
 struct requires_impl {
-  using template_error_types_failed_to_meet_requirements_on_template_parameters =
+  using template_error_types_failed_to_meet_requirements_on_template_parameters=
```

Comment:
Not sure, did not check clang-format source code. I can look and see if it's possible to make a patch to submit to them if you like.